configs/dc2.yaml

```diff
@@ -0,0 +1,51 @@
+facility_name: Rubin-LSST
+obs_collection: LSST.DC2
+collections: ["2.2i/runs/DP0.1"]
+use_butler_uri: false
+dataset_types:
+  raw:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.raw
+    calib_level: 1
+    obs_id_fmt: "{records[exposure].obs_id}"
+    o_ucd: phot.count
+    access_format: image/fits
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+  calexp:
```

Comment:
I think diffims are missing here but are available in the DC2/DP0.2 Butler?

Reply:
I can't seem to find any dataset type that resembles diffim in `2.2i/runs/DP0.1` collection. Here is the list of dataset types in that collection:
```
bias
cal_ref_cat_2_2
calexp
calexpBackground
camera
coaddDriver_config
dark
deepCoadd
deepCoadd_calexp
deepCoadd_calexp_background
deepCoadd_deblendedFlux
deepCoadd_deblendedFlux_schema
deepCoadd_deblendedModel_schema
deepCoadd_det
deepCoadd_det_schema
deepCoadd_forced_src
deepCoadd_forced_src_schema
deepCoadd_mcalmax_deblended
deepCoadd_mcalmax_deblended_schema
deepCoadd_meas
deepCoadd_measMatch
deepCoadd_measMatchFull
deepCoadd_meas_schema
deepCoadd_mergeDet
deepCoadd_mergeDet_schema
deepCoadd_nImage
deepCoadd_ngmix_deblended
deepCoadd_ngmix_deblended_schema
deepCoadd_peak_schema
deepCoadd_ref
deepCoadd_ref_schema
flat
icSrc
icSrc_schema
multiBandDriver_config
packages
raw
singleFrameDriver_config
sky
skyCorr
skyCorr_config
skyMap
src
srcMatch
src_schema
```

Reply:
I was thinking that was going to be the case. I don't think dp0.2 DC2 dataset has any time variability data.

Reply:
I thought I'd understood that they have supernovae and are running AP.  Maybe not in this particular test dataset?  Actually I see from the collection name that you are looking at **DP0.1** anyway, not DP0.2!

We definitely need to test this on DP0.2 output - it's not the same as DP0.1 (which is a Gen2->3 conversion of an old, externally executed production).

Reply:
If I knew where to find DP0.2. I looked at the collections in dc2 repo but nothing really obvious that I could see.

Reply:
On data-int.lsst.cloud:
```
$ butler query-dataset-types dp02 | grep -i diff
                                               deepDiff_warpedExp
                                forcedPhotDiffOnDiaObjects_config
                                   forcedPhotDiffOnDiaObjects_log
                              forcedPhotDiffOnDiaObjects_metadata
                                          forcedPhotDiffim_config
                                             forcedPhotDiffim_log
                                        forcedPhotDiffim_metadata
                                                      forced_diff
                                            forced_diff_diaObject
                                     forced_diff_diaObject_schema
                                               forced_diff_schema
                                  goodSeeingDiff_assocDiaSrcTable
                                       goodSeeingDiff_diaObjTable
                                            goodSeeingDiff_diaSrc
                                       goodSeeingDiff_diaSrcTable
                                     goodSeeingDiff_diaSrc_schema
                                     goodSeeingDiff_differenceExp
                                   goodSeeingDiff_fullDiaObjTable
                                       goodSeeingDiff_templateExp
                                         goodSeeingDiff_warpedExp
                                           imageDifference_config
                                              imageDifference_log
                                         imageDifference_metadata
                                 plot_E1Diff_scatter_visit_config
                                    plot_E1Diff_scatter_visit_log
                               plot_E1Diff_scatter_visit_metadata
                                     plot_E1Diff_sky_visit_config
                                        plot_E1Diff_sky_visit_log
                                   plot_E1Diff_sky_visit_metadata
                                 plot_E2Diff_scatter_visit_config
                                    plot_E2Diff_scatter_visit_log
                               plot_E2Diff_scatter_visit_metadata
                                     plot_E2Diff_sky_visit_config
                                        plot_E2Diff_sky_visit_log
                                   plot_E2Diff_sky_visit_metadata
                                  plot_ShapeDiff_sky_visit_config
                                     plot_ShapeDiff_sky_visit_log
                                plot_ShapeDiff_sky_visit_metadata
                          plot_shapeSizeDiff_scatter_visit_config
                             plot_shapeSizeDiff_scatter_visit_log
                        plot_shapeSizeDiff_scatter_visit_metadata
                                   scatterPlotVisit_E1Diff_ixxPSF
                           scatterPlotVisit_E1Diff_ixxPSF_scatter
                                   scatterPlotVisit_E2Diff_ixxPSF
                           scatterPlotVisit_E2Diff_ixxPSF_scatter
                                scatterPlotVisit_shapeDiff_ixxPSF
                        scatterPlotVisit_shapeDiff_ixxPSF_scatter
                                       skyPlotVisit_E1Diff_ixxPSF
                                   skyPlotVisit_E1Diff_ixxPSF_sky
                                       skyPlotVisit_E2Diff_ixxPSF
                                   skyPlotVisit_E2Diff_ixxPSF_sky
                                           skyPlot_Ediff_quiver_i
```

Reply:
I never used `data-int.lsst.cloud` and it looks like I am not welcome there:
```
Authentication failed with the following error:
Error: User unauthorized: andy-slac is not a member of any authorized groups
To report problems or ask for help, please open an issue in the GitHub rubin-dp0/Support project. 
```
I'm not sure opening a ticket on GitHub is the right way to ask for an access?

Reply:
Looking at the `dp02` repo on `data-int`, I think that the latest DP0.2 collection there is 
"2.2i/runs/DP0.2/v23_0_2/PREOPS-905/step7". And I cannot find any of the `*diff*` dataset types in that collection.

Reply:
You will find DP0.2 difference images in data-int collection `2.2i/runs/DP0.2/v23_0_1/PREOPS-905/step4`, dataset type `goodSeeingDiff_differenceExp`.

Reply:
@gpdf, I made one more CSV file which includes `goodSeeingDiff_differenceExp` dataset type: https://lsst.ncsa.illinois.edu/~salnikov/dp02.2.2i.runs.test-med-1.v23_0_0_rc2.PREOPS-863.csv.gz
It is relatively small ~88k rows, made from `2.2i/runs/test-med-1/v23_0_0_rc2/PREOPS-863` collection.